### PR TITLE
docs: Add documentation for RocketChat

### PR DESCRIPTION
**Pull request by sergeyshevch**
Signed-off-by: Sergey Shevchenko <sergey.shevchenko@dteam.dev>

### CODE REVIEW

File: docs/services/rocketchat.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+## Configuration
```
Comment (ryota-sakamoto) [inline docs/services/rocketchat.md:1]: Would you add Parameters section like as follows, please?

https://github.com/argoproj/notifications-engine/blob/aa774e755de808339b9790c995ce8fd14120f003/docs/services/github.md#L3-L11

File: docs/services/rocketchat.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+8. Finally, use these credentials to configure the RocketChat integration in the `argocd-configmap` config map: 
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <config-map-name>
+data:
+  service.rocketchat: |
+    alias: <alias>          # Username alias
```
Comment (ryota-sakamoto) [inline docs/services/rocketchat.md:36]: Would you define minimum sample, please? and move to Parameters section.

File: docs/services/rocketchat.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    notifications.argoproj.io/subscribe.on-sync-succeeded.rocketchat: #my_channel
+```
+
+## Templates
+
+Notification templates can be customized with RocketChat [attachments](https://developer.rocket.chat/api/rest-api/methods/chat/postmessage#attachments-detail).
+
+*Note: Attachments structure in Rocketchat is same with Slack attachments [feature](https://api.slack.com/messaging/composing/layouts).*
+
+<!-- TODO: @sergeyshevch Need to add screenshot with RocketChat attachments -->
```
Comment (ryota-sakamoto) [inline docs/services/rocketchat.md:62]: please paste screenshot.

File: docs/services/rocketchat.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  service.rocketchat: |
+    alias: <alias>          # Username alias
+    email: $rocketchat-email
+    password: $rocketchat-password
+    icon: <emoji>           # Emoji for using as avatar
+    avatar: <avatarUrl>     # optional
+    serverUrl: <url>        # Your RocketChat instance URL
+```
+
+9. Create a subscription for your Slack integration:
```
Comment (ryota-sakamoto) [inline docs/services/rocketchat.md:44]: Is it `Rocket.Chat`?

**Review by ryota-sakamoto — CHANGES_REQUESTED** [review]
please add to `Service Types`.
https://github.com/argoproj/notifications-engine/blob/aa774e755de808339b9790c995ce8fd14120f003/docs/services/overview.md#L39-L49

**Comment by alexmt** [comment]
Closing since https://github.com/argoproj/notifications-engine/pull/25 has the same changes

**Pull request closed without merging**